README.adoc

```diff
@@ -104,11 +104,7 @@ https://docs.antora.org/antora/1.0/install/troubleshoot-nodegit/
 ./bin/build.sh local-site.yml
 ----
 + 
-After building for the first time, you can use the following command for subsequent builds:
-+
-----
-./bin/build.sh local-site.yml
```

Comment:
@finp I removed this part because the command line is the same. Maybe it was only a typo and it was intended to be `quick-build.sh` instead of `build.sh`?